lib/schedule.js

```diff
@@ -327,7 +341,7 @@ Range.prototype.contains = function(val) {
 
   NOTE: Cron months are 1-based, but RecurrenceRule months are 0-based.
 */
-function RecurrenceRule(year, month, date, dayOfWeek, hour, minute, second) {
+function RecurrenceRule(year, month, date, dayOfWeek, hour, minute, second, timezone) {
   this.recurs = true;
```

Comment:
Still it's still a little weird to me having the timezone in the `RecurrenceRule` constructor. If you want to follow this route I would remove the `timezone` argument, so if a user wants to schedule a job in a specific timezone using RR won't have 2 different places where timezone could be set.
```js
const rule = new RecurrenceRule(2017, 8, 21);
const job = schedule.scheduleJob({rule: rule, timezone: 'Asia/Kuala_Lumpur'}
```

Also, another pro for attaching the timezone to the `Job` is that it could be useful asking the `Job` in which timezone it's running,